amplfi/train/priors.py

```diff
@@ -27,12 +27,103 @@ def sg_prior() -> ParameterSampler:
     )
 
 
+def precessing_to_lalsimulation_parameters(
+    parameters: dict[str, torch.Tensor]
+) -> dict[str, torch.Tensor]:
+    """
+    Convert precessing spin parameters to lalsimulation parameters
```

Comment:
Can we add a test here to test to check batched input?

Reply:
For which part? The `bilby_to_lalsim_spins` and `chirp_mass_and_mass_ratio_to_components` have tests on the ml4gw side already that test for batched input

Reply:
I see. (Commented without checking that). All good then.